unittestsUS21.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+import examples
+from project03 import ensureMarriageGenderRoles
+
+class TestMarriageGendersVerification(unittest.TestCase):
```

Comment:
Usual standard is "MarriageGendersTestCase", where `TestCase` is always a suffix of a unit test class